src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -115,10 +117,20 @@ const SidebarLinks = (props) => {
     return (
         <View style={[styles.flex1, styles.h100, {marginTop: props.insets.top}]}>
             <View style={[chatSwitcherStyle]}>
-                <ChatSwitcherView
-                    onLinkClick={props.onLinkClick}
-                    isChatSwitcherActive={props.isChatSwitcherActive}
-                />
+                {!props.isChatSwitcherActive ? (
+                    <TouchableOpacity
+                        style={[styles.flexRow, styles.sidebarHeaderTop]}
+                        onPress={() => ChatSwitcher.show()}
+                    >
+                        <MagnifyingGlassIcon width={30} height={30} />
+                    </TouchableOpacity>
```

Comment:
This can be moved in between these two component here:

https://github.com/Expensify/Expensify.cash/blob/91fbf6cb1ce5451f1530182ce98ef2dc5e9af664/src/pages/home/sidebar/SidebarLinks.js#L142-L143

Reply:
If it is moved, there will be an issue like below.
![Screen Shot 2021-01-14 at 4 08 32 PM](https://user-images.githubusercontent.com/74012504/104655239-0d5f8380-5683-11eb-8f66-c3a1bd843312.png)
![Screen Shot 2021-01-14 at 4 11 09 PM](https://user-images.githubusercontent.com/74012504/104655309-26683480-5683-11eb-8913-2b06006515c4.png)

Reply:
If you push your changes I will try to help resolve.

Reply:
pushed